pom.xml

```diff
@@ -127,7 +127,7 @@ under the License.
 		<flink.markBundledAsOptional>true</flink.markBundledAsOptional>
 		<target.java.version>11</target.java.version>
 		<slf4j.version>1.7.36</slf4j.version>
-		<log4j.version>2.17.1</log4j.version>
+		<log4j.version>2.23.1</log4j.version>
```

Comment:
Any reason why not bump to 2.24.1, which is the latest ATM?

Reply:
I have no idea what that is cuz i dunno java.

Reply:
I'm sorry, I tested these changes long back (about a month back) and there are couple of new versions now. Let me push the new changes 🙂

Reply:
@ferenc-csaky @Juoelenis can you re-review the PR.

Reply:
Thanks, no worries!